modules/gdscript/gdscript_utility_functions.cpp

```diff
@@ -124,6 +124,14 @@ struct GDScriptUtilityFunctionsDefinitions {
 		*r_ret = String(result);
 	}
 
+	static inline void ord(Variant *r_ret, const Variant **p_args, int p_arg_count, Callable::CallError &r_error) {
+		DEBUG_VALIDATE_ARG_COUNT(1, 1);
+		DEBUG_VALIDATE_ARG_TYPE(0, Variant::STRING);
+		String string = *p_args[0];
```

Comment:
This implementation allows implicit conversions, such as `ord(1) == ord("1")`. I think this may be confusing. Using `VariantInternal` may be more appropriate - or at least convert the argument strictly (`can_convert_strict`).

Reply:
Hrm, I guess `DEBUG_VALIDATE_ARG_TYPE` already does the check exactly like that, but it's not in release build. That's _probably_ acceptable.